doc/docs/overview_and_ui.md

```diff
@@ -4,22 +4,66 @@ After starting nRF Connect Bluetooth® Low Energy, the application window is dis
 
 ![Application window](./screenshots/nRF_connect_app_window.png "Application window")
 
-It consists of the following main elements.
+The window consists of the following main elements.
 
 ## Select Device
 
 To the upper left, you can select a local serial device. You can use the selected local device to scan for Bluetooth Low Energy devices, which are listed in the side panel below.
 
-You can switch between the following application tabs, using the navigation bar.
+You can access the following options by selecting the arrow under the device name.
 
-## Connection Map
+- Show the device at the top of the list by making it favorite.
+- Rename the device.
+- View and select between device Com Ports.
```

Comment:
“Com Ports” is strange. 
“COM ports” is better. 
“serial ports” is probably the best.